tests/units/array/test_array.py

```diff
@@ -493,7 +492,24 @@ def test_parameterize_list():
     from docarray import DocList, BaseDoc
 
     with pytest.raises(TypeError) as excinfo:
-        doc = DocList[BaseDoc()]
-        assert doc is None
+        da = DocList[BaseDoc()]
+        assert da is None
 
     assert str(excinfo.value) == 'Expecting a type, got object instead'
+
+
+def test_not_double_subcriptable():
+    from docarray import DocList, DocVec
+    from docarray.documents import TextDoc
+
+    with pytest.raises(TypeError) as excinfo:
+        da = DocList[TextDoc][TextDoc]
+        assert da is None
+
+    assert 'not subscriptable' in str(excinfo.value)
+
+    with pytest.raises(TypeError) as excinfo:
```

Comment:
I don't know if I am allowed to comment on PR but this looks like duplicated code

Reply:
this is on tests, nothing wrong about that